src/main/java/com/ita/if103java/ims/service/impl/AssociateServiceImpl.java

```diff
@@ -67,6 +67,17 @@ public AssociateDto view(Long id) {
         return associateDto;
     }
 
+    @Override
+    public List<AssociateDto> findAll() {
+        List<AssociateDto> associateDtos = associateDtoMapper.toDtoList(associateDao.findAll());
+
+        for (AssociateDto associateDto : associateDtos) {
+            associateDto.setAddressDto(addressDtoMapper.toDto(addressDao.findByAssociateId(associateDto.getId())));
+        }
```

Comment:
maybe, will be better to use list.foreach() here

Reply:
![image](https://user-images.githubusercontent.com/33421961/70570113-f27b2f80-1ba3-11ea-8c3f-390bc789f369.png)
The last one is the fastest)